ssclj/jar/build.boot

```diff
@@ -156,7 +156,8 @@
                             #"config-hsqldb.edn"
                             #"log4j.properties"}
                  :invert true)
-           (aot :namespace #{'com.sixsq.slipstream.ssclj.app.main 'com.sixsq.slipstream.ssclj.usage.summarizer})
+           (aot :namespace #{'com.sixsq.slipstream.ssclj.app.main
+                             'com.sixsq.slipstream.ssclj.usage.summarizer})
```

Comment:
Is the usage summarizer used in the metering service?  If so, why does it need to be AOT compiled?

Reply:
Actually this file was accidentally touched, it should not be changed from its original content in master